ga4gh/datamodel/variants.py

```diff
@@ -123,6 +124,7 @@ def addCallSet(self, sampleName):
         self._callSetIdMap[callSetId] = callSet
         self._callSetNameMap[sampleName] = callSet
         self._callSetIds.append(callSetId)
+        self._callSets[callSet.getId()] = len(self._callSetIds) - 1
```

Comment:
The name _callSets implies that this contains CallSet objects and is confusing. maybe _callSetIdToIndex?